models/articles.model.js

```diff
@@ -23,7 +23,31 @@ exports.getAllArticles = () => {
         ORDER BY created_at DESC;`
     )
     .then((data) => {
-      console.log(data.rows)
       return data.rows;
     });
 };
+
+exports.insertCommentByArticleId = (article_id, username, body) => {
+return db.
+query(`SELECT * FROM articles
+     WHERE articles.article_id = $1
+`, [article_id]).
+then((data) => {
+  if (!data.rows.length){
+    return Promise.reject({ status: 404, msg: "article does not exist" });
+  }
+ 
+  else {
```

Comment:
dont need the else here because if the previous condition is true you return and thus terminate the function